examples/cluster.yaml

```diff
@@ -56,6 +56,8 @@ spec:
     - hostname: "worker-1"
 
   azure:
+    keyVault:
+      vaultName: "example-cluster-vault"
```

Comment:
Here is the new `azuretpr` value.

Reply:
We decided to it's `Name` in the tpr project, so probably should be `name` here.